app/views/users/show.html.haml

```diff
@@ -0,0 +1,44 @@
+%header.entry-header
+  - if @user.admin?
+    %h1.entry-title.is-admin=  @user.email
+  - else
+    %h1.entry-title=  @user.email
+
+
+.entry-content
+  %p
+    %b #{t('.email')}:
+    = @user.email
+
+  - if @user.last_sign_in_at.blank? && @user.current_sign_in_at.blank?
+    %p  #{@user.email} #{t('.user_has_never_signed_in')}
+
+  - else
+    %p
+      %b #{t('.last_login')}:
+      #{ most_recent_login_time @user}
+      (#{time_ago_in_words(most_recent_login_time @user)} ago)
+    %p
+      %b #{t('.last_login_ip')}:
+      #{ most_recent_login_ip @user}
+
+
+
+    %p
+      %b #{t('.logged_in_count')}:
+      = @user.sign_in_count
+
+  - if @user.admin?
+    %p.is-admin #{t('.is_an_admin')}
```

Comment:
Thanks.  That's helpful. 

I keep struggling with this kind of thing in HAML.  (@thesuss has helped me with this same syntax issue many times.)  Time for me to go read thru the HAML specs and practice with some tutorials. :-)